### PR TITLE
Brooks jeffrey regions in IN, KS, and MS that don't require special casing

### PR DESCRIPTION
**Pull request by terinpw**

### CODE REVIEW

File: recidiviz/ingest/scraper_template/region_scraper_test.txt [hunk 1]
```diff
@@ -21,7 +21,6 @@ from lxml import html
 
 from recidiviz.ingest import constants
 from recidiviz.ingest.models.ingest_info import IngestInfo
-from recidiviz.ingest.task_params import Task
```
Comment (jamwalla) [inline recidiviz/ingest/scraper_template/region_scraper_test.txt]: hmm, we should keep this for tests that actually do stuff but this is something to consider for #803

File: recidiviz/ingest/__init__.py [hunk 2]
```diff
@@ -56,10 +56,18 @@
...
+import recidiviz.ingest.us_ks_cherokee
+import recidiviz.ingest.us_ks_jefferson
+import recidiviz.ingest.us_ks_pratt
+import recidiviz.ingest.us_mo_stone
+import recidiviz.ingest.us_ms_clay
+import recidiviz.ingest.us_ms_kemper
+import recidiviz.ingest.us_ms_tunica
 import recidiviz.ingest.us_ms_desoto
 import recidiviz.ingest.us_mt_gallatin
-import recidiviz.ingest.us_mo_stone
```
Comment (jamwalla) [inline recidiviz/ingest/__init__.py]: oh no!

Comment (jamwalla) [reply recidiviz/ingest/__init__.py]: oh yes! it's alphabetized

File: recidiviz/ingest/scraper_template/region_scraper_test.txt (hunk 1 above)
Comment (terinpw) [reply recidiviz/ingest/scraper_template/region_scraper_test.txt]: Good catch -- reverted!

File: recidiviz/ingest/__init__.py (hunk 2 above)
Comment (terinpw) [reply recidiviz/ingest/__init__.py]: I just alphabetized it! it's still there.

**Comment by jessex** [comment]
Goodness, @terinpw

**Pull request merged**